DEVELOPMENT.PERCONA.md

```diff
@@ -0,0 +1,46 @@
+# Into
+
+This guide is a step-by-step instruction that helps you with local development environment.
+
+# Setup
+
+Follow these steps:
+
+1: run docker images with PMM and MySQL (in foreground mode):
+
+```shell
+docker-compose up
+```
+
+2: in separate terminal:
+
+```shell
+yarn start
```

Comment:
Do we need to run `yarn install` before `yarn start`?

Reply:
I think we need `yarn install`

Reply:
updated